### ISSUE TITLE
Feature request: Thumbnail Engine for supported formats

### ISSUE THREAD

**Issue by microraptor** [projectM-visualizer/projectm#636]
**Is your feature request related to a problem? Please describe.**
When browsing presets, it is hard to remember names and the only way to see waht they look like is to put them in the presets folder and then find them inside projectM. Some preset packs come with JPG previews, but those are in the minority.

**Describe the solution you'd like**
It would be great, if projectM could be used as a thumbnail generator for the different OS.

**Describe alternatives you've considered**
Perhaps its better to have a different program/repo for the thumbnail generator for each OS.

**Additional context**
Thank you for this awesome project!

**Comment by kblaschke** [comment]
Yes, that's a great idea! Won't be something the core library will do, but a small tool generating random audio data and generating an image of each preset shouldn't be too hard to implement.
